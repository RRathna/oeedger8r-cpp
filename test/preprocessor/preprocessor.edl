// Copyright (c) Open Enclave SDK contributors.
// Licensed under the MIT License.

enclave {
  trusted
  {
#ifdef TEST
    public void first_ecall();
#else
    public void second_ecall();
#endif
  };
};
