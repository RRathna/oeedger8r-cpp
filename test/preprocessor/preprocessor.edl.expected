enclave {
  trusted
  {
    public void first_ecall();
  };
};
